Blog-template created with additional Navbar feature

Tushar98644 opened the pull request.

shelcia commented:
Please don't use this code

    const show1 = () => {
        document.getElementById('h1').style.opacity = "0";
        document.getElementById('h2').style.opacity = "0";
        document.getElementById('h3').style.opacity = "0";
        document.getElementById('h4').style.opacity = "0";
        document.getElementById('h5').style.opacity = "0";
        document.getElementById('h6').style.opacity = "0";
        document.getElementById('heading_2').style.opacity = "0.7";
        document.getElementById('blog').style.textDecoration = "none";
        document.getElementById('about').style.textDecoration = "underline";




    }
    const show2 = () => {
        document.getElementById('heading_2').style.opacity = "0";
        document.getElementById('h1').style.opacity = "1";
        document.getElementById('h2').style.opacity = "1";
        document.getElementById('h3').style.opacity = "1";
        document.getElementById('h4').style.opacity = "1";
        document.getElementById('h5').style.opacity = "1";
        document.getElementById('h6').style.opacity = "1";
        document.getElementById('blog').style.textDecoration = "underline";
        document.getElementById('about').style.textDecoration = "none";


and why are you adding bootstrap script in the bottom

Tushar98644 commented:
updated

> Please don't use this code
> 
> ```
> const show1 = () => {
>     document.getElementById('h1').style.opacity = "0";
>     document.getElementById('h2').style.opacity = "0";
>     document.getElementById('h3').style.opacity = "0";
>     document.getElementById('h4').style.opacity = "0";
>     document.getElementById('h5').style.opacity = "0";
>     document.getElementById('h6').style.opacity = "0";
>     document.getElementById('heading_2').style.opacity = "0.7";
>     document.getElementById('blog').style.textDecoration = "none";
>     document.getElementById('about').style.textDecoration = "underline";
> 
> 
> 
> 
> }
> const show2 = () => {
>     document.getElementById('heading_2').style.opacity = "0";
>     document.getElementById('h1').style.opacity = "1";
>     document.getElementById('h2').style.opacity = "1";
>     document.getElementById('h3').style.opacity = "1";
>     document.getElementById('h4').style.opacity = "1";
>     document.getElementById('h5').style.opacity = "1";
>     document.getElementById('h6').style.opacity = "1";
>     document.getElementById('blog').style.textDecoration = "underline";
>     document.getElementById('about').style.textDecoration = "none";
> ```
> 
> and why are you adding bootstrap script in the bottom

updated

Tushar98644 commented:
> Please don't use this code
> 
> ```
> const show1 = () => {
>     document.getElementById('h1').style.opacity = "0";
>     document.getElementById('h2').style.opacity = "0";
>     document.getElementById('h3').style.opacity = "0";
>     document.getElementById('h4').style.opacity = "0";
>     document.getElementById('h5').style.opacity = "0";
>     document.getElementById('h6').style.opacity = "0";
>     document.getElementById('heading_2').style.opacity = "0.7";
>     document.getElementById('blog').style.textDecoration = "none";
>     document.getElementById('about').style.textDecoration = "underline";
> 
> 
> 
> 
> }
> const show2 = () => {
>     document.getElementById('heading_2').style.opacity = "0";
>     document.getElementById('h1').style.opacity = "1";
>     document.getElementById('h2').style.opacity = "1";
>     document.getElementById('h3').style.opacity = "1";
>     document.getElementById('h4').style.opacity = "1";
>     document.getElementById('h5').style.opacity = "1";
>     document.getElementById('h6').style.opacity = "1";
>     document.getElementById('blog').style.textDecoration = "underline";
>     document.getElementById('about').style.textDecoration = "none";
> ```
> 
> and why are you adding bootstrap script in the bottom
